src/local.js

```diff
@@ -46,6 +47,24 @@ function cachedToken(scope) {
 		});
 }
 
+function adjustClockSkew(res) {
+	var dateHeader = res.headers && res.headers.date;
+	if (!dateHeader) {
+		return;
+	}
+
+	var serverTime = new Date(dateHeader).getTime();
+	if (serverTime !== serverTime) {
```

Comment:
Neat! Is there another possible result that would not equate? Otherwise it might be clearer to write
`if (serverTime !== NaN)`